test_goldens/testing/util.dart

```diff
@@ -114,10 +114,11 @@ Future<void> setUpBinding(
   Size size = mobileSize,
   Brightness brightness = Brightness.light,
 }) async {
-  tester.binding.window.physicalSizeTestValue = size;
-  tester.binding.window.devicePixelRatioTestValue = 1.0;
-  tester.binding.window.platformDispatcher.textScaleFactorTestValue = 1.0;
-  tester.binding.window.platformDispatcher.platformBrightnessTestValue =
+  tester.view.physicalSize = size;
+  tester.view.devicePixelRatio = 1.0;
+  tester.view.platformDispatcher.textScaleFactorTestValue = 1.0;
+  tester.view.platformDispatcher.platformBrightnessTestValue =
```

Comment:
There should be a `tester.platformDispatcher` property to use for these.

Reply:
Done.